resources/geocoding/en/52.txt

```diff
@@ -0,0 +1,418 @@
+# Copyright (C) 2017 Pindrop Security, Inc.
+
+# Licensed under the Google Software Grant and Corporate Contributor License Agreement (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# https://cla.developers.google.com/clas/new?domain=DOMAIN_GOOGLE&kind=KIND_CORPORATE
+
+# Generated from:
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200008A0002PDFS.pdf
+# Internal statistics data (2015-07-10).
+# In cases of multiple areas for same number:
+# 	If all areas in same state, default to state
+# 	If areas in different states, default to largest population and
+# 	  name both states
+
+52223|Puebla
+52224|Puebla
+52225|Tlapacoyan, VER
+52226|Altotonga/Jalacingo, VER
+52227|Huejotzingo/San Buenaventura Nealtican, PUE
+52228|Jalapa/Tuzamapan, VER
+52229|Veracruz, VER
+52231|Teteles/Teziutlán, PUE
+52232|La Vigueta/Martínez de la Torre, VER
+52233|Puebla
+52235|Veracruz
+52236|Teotitlán del Camino, OAX/PUE
```

Comment:
I think Pueblo/Oaxaca is the best you can do.

Alternatively you could do Ajalpan/Calipan PUE, Cuicatlan/Huautla de Jimenez/Teotitlan del Camino OAX
but it's not concise.

Third option: list the municipalities only, not the poblacion; e.g. Calipan is in Coxcatlan.

Fourth option: use the Mexican detailed information and add more prefixes:
https://sns.ift.org.mx:8081/sns-frontend/planes-numeracion/descarga-publica.xhtml

But I don't know what the "serie" column is, not sure if you can just do NIR + first digit of numeracion inicial, NIR + first digit of numeracion final, and all in between....

Anyway it has more detailed data, but requires more processing. We do have something that could help compress it afterwards: http://libphonenumber-combine-geodata.appspot.com/ - but it can be a bit overaggressive.